lldb/source/Commands/Options.td

```diff
@@ -216,12 +216,6 @@ let Command = "breakpoint set" in {
   def breakpoint_set_file_colon_line : Option<"joint-specifier", "y">, Group<12>, Arg<"FileLineColumn">,
     Required, Completion<"SourceFile">,
```

Comment:
I'm a bit confused about this change — does that mean the option doesn't (and didn't) show up in the online help for the breakpoint command? Is that what we want?

Reply:
I updated the PR description to point out that this commented-out definition is a duplicate. The actual definition of `breakpoint_set_file_colon_line` is a few lines up. Maybe this duplicate came from a merge?